samples/quickstart.js

```diff
@@ -14,42 +14,51 @@
 
 'use strict';
 
-// [START speech_quickstart]
 async function main() {
```

Comment:
Should this just be function main()? I think we are still bubbling up the correct error on lines 59 - 62.

Reply:
Yeah good call.  I switched this up in a few places, and hopefully cleaned up the failing tests.